scripts/test/get_all_tests.sh

```diff
@@ -8,6 +8,8 @@ SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 declare -a test_array
 for test_file in "$SCRIPT_DIR"/../../test/test_*.py; do
   test_name="$(basename "$test_file")"
-  test_array=("${test_array[@]}" "$test_file" "$test_name")
+  if [[ ! $test_name =~ .*gpu.* ]]; then
+    test_array=("${test_array[@]}" "$test_file" "$test_name")
+  fi
```

Comment:
Not nice, but I propose to improve this when we migrate the integration tests to pytest.